rohmu/object_storage/azure.py

```diff
@@ -8,15 +8,26 @@
 
 from __future__ import annotations
 
-from ..common.models import ProxyInfo, StorageModel
-from ..common.statsd import StatsdConfig
-from ..notifier.interface import Notifier
-from ..typing import Metadata
-from .base import IncrementalProgressCallbackType, ProgressProportionCallbackType
-
 # pylint: disable=import-error, no-name-in-module
 from azure.core.exceptions import HttpResponseError, ResourceExistsError
 from azure.storage.blob import BlobServiceClient, ContentSettings
+from rohmu.common.statsd import StatsdConfig
+from rohmu.errors import FileNotFoundFromStorageError, InvalidConfigurationError, StorageError
+from rohmu.notifier.interface import Notifier
+from rohmu.object_storage.base import (
+    BaseTransfer,
+    IncrementalProgressCallbackType,
+    IterKeyItem,
+    KEY_TYPE_OBJECT,
+    KEY_TYPE_PREFIX,
+    ProgressProportionCallbackType,
+)
+from rohmu.object_storage.config import (  # pylint: disable=unused-import
+    AZURE_MAX_BLOCK_SIZE as MAX_BLOCK_SIZE,
+    AzureObjectStorageConfig as Config,
+    calculate_azure_max_block_size as calculate_max_block_size,
+)
+from rohmu.typing import Metadata
```

Comment:
Changes like these is there result of moving the lines back and forth with PyCharm refactoring tool. Absolute imports should be preferred, so I see no harm here. I can create a separate PR fixing all imports if needed.